lib/custom_widgets/movie_overview.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+import '../text_styles/custom_text_style.dart';
+import '../ui_constants.dart';
+
+class MovieOverview extends StatelessWidget {
+  const MovieOverview({
+    super.key,
+    required this.overview,
+  });
+
+  final String overview;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(
+        vertical: MovieDetailsUiConstants.overviewPadding,
+        horizontal: MovieDetailsUiConstants.releaseDateRowPadding,
+      ),
+      child: Positioned(
```

Comment:
I'd move this `Positioned` to the `PosterAndOverviewStack`, to leave this widget not depending on that positioning, thus more generic and reusable

Reply:
done